server.go

```diff
@@ -1116,7 +1116,7 @@ func (s *Server) DeriveSweepAddr(inputs []input.Input,
 	change lnwallet.AddrWithKey) lfn.Result[sweep.SweepOutput] {
```

Comment:
ultranit: Go doc comment is slightly confusing to me. Shouldn't this read: "and maybe results in an extra sweep output" instead of "and maybe results an extra sweep output"? Same here: 
https://github.com/lightninglabs/taproot-assets/blob/a50c87ab4afade81ee75d59dfac64095479698c4/tapchannel/aux_sweeper.go#L1496-L1497